pipelines/install/03-tests.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: run tests
+  hosts:
+    - "{{ forklift_server_name }}"
+  become: true
+  vars_files:
+    - ../vars/install_base.yml
+    - ../vars/forklift_{{ pipeline_type }}.yml
+  roles:
+    - pulp_health_check
```

Comment:
We apply become on our specific tasks. You shouldn't set it for the entire role.